launch_crab_multithreading.py

```diff
@@ -1052,14 +1539,18 @@ def submit(config, dryrun = False):
         # JSON filter
         jsonName = ""
         if is2016:
-            jsonName = "Cert_271036-284044_13TeV_23Sep2016ReReco_Collisions16_JSON"
+            #jsonName = "Cert_271036-284044_13TeV_23Sep2016ReReco_Collisions16_JSON"
+            jsonName = "Cert_271036-284044_13TeV_ReReco_07Aug2017_Collisions16_JSON.txt"
```

Comment:
Just to be sure: The miniAOD ntuplizer is _not_ using the value set here (`jsonName` is set directly there, not considering the parsed options) and the value currently used there is the one older one – now commented out here. Is that the intended behavior @MelanieEich?

Reply:
More importantly, the JSON to be used for crab submission is actually set below (where also the older 2016 file is used). So `jsonName` is left unused both locally (for the miniAOD ntuplizer) _and_ by crab.